### ISSUE TITLE
Reproduce results of fitted model

### ISSUE THREAD

**Issue by rusad** [benfred/implicit#36]
Every time when i run model.fit() i obtain different values in model.user_factors and model.item_factors. How can i get reproducible result after model fitting?

**Comment by benfred** [comment]
The initial factors are randomly assigned - and the output will be different every time afterwards (though the overall loss should converge to the same value).

To remove variability - the unittests for that function seed the RNG with a fixed value: https://github.com/benfred/implicit/blob/master/tests/als_test.py#L65 .

**Comment by rusad** [comment]
Ok, thanks
